auslib/web/common/rules_spec.yml

```diff
@@ -0,0 +1,124 @@
+paths:
+  /rules:
```

Comment:
Looks good, I'll working more on balrog next saturday(3).
About the new endpoints implementations living in auslib.web.common, it's ok?

https://github.com/ex-Dev/balrog/blob/38c7cb84489664d5df379c26a46fc8e186a34255/auslib/web/common/rules.py#L13

https://github.com/ex-Dev/balrog/blob/38c7cb84489664d5df379c26a46fc8e186a34255/auslib/web/common/releases.py#L14